### PR TITLE
DMP-58 Connecting DARTS API to Azure Application Insights hosted in Darts Shared Infrastructure

### PR DESCRIPTION
**Pull request by alancoleman**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DMP-58

### Change description ###
Initial commit of secrets to allow for connection to Azure App Insights. There is no real consistency across the HMCTS Repos on how this is achieved.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: lib/applicationinsights.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "connectionString": "${file:/mnt/secrets/darts/AppInsightsConnectionString}",
```
Comment (alancoleman) [inline lib/applicationinsights.json:2]: I pretty sure this is correct.

File: lib/applicationinsights.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
   "role": {
     "name": "rpe-demo"
```
Comment (alancoleman) [inline lib/applicationinsights.json:4]: I can't find any consistent way of naming a role in the other HMCTS Repos.

**Comment by alancoleman** [comment]
https://learn.microsoft.com/en-us/azure/azure-monitor/app/java-standalone-config#configuration-file-path

**Comment by davet1985** [comment]
Superseded

**Pull request closed without merging**